### PR TITLE
Bids 2858/notifications management dashboard table

### PR DESCRIPTION
**Pull request by MauserBitfly**
This PR:
- creates the notifications management dashboard table
- adds the table to the management modal
- moves the management modal file/translations

This PR does not:
- use the 'real' API / types (as they don't exist yes)
- create and open edit dialogs 
- implement the delete / reset function

### CODE REVIEW

File: frontend/stores/notifications/useNotificationsManagementDashboardStore.ts
Comment (D13ce) [inline frontend/stores/notifications/useNotificationsManagementDashboardStore.ts:1]: **Question:** As discussed, I am not sure whether this should be a store, a composable or something else.
Notifications are always private so accessing and rendering the data on the server does not provide any real benefits, right?

Just as a note: Keep in mind that we will have several tabs in this modal, all of them storing data and all of them having their own endpoints to `GET`/`PUT` the data. Maybe this should be considered, maybe it does not matter now.

File: frontend/components/notifications/management/NotificationsManagementDashboards.vue [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+          header-class="dashboard-col"
+          :sortable="true"
+          :header="$t('notifications.col.dashboard')"
+        >
+          <template #body="slotProps">
+            <span>
+              {{ slotProps.data.dashboard_name }}
+            </span>
+          </template>
+        </Column>
```
Comment (D13ce) [inline frontend/components/notifications/management/NotificationsManagementDashboards.vue:124]: **Bug:** Compared to the **Design**,
image
the **Implementation** is missing the dashboard type icon
image

File: frontend/components/notifications/management/NotificationsManagementDashboards.vue [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    @include utils.set-all-width(340px);
+
+    @media (max-width: 1360px) {
+      @include utils.set-all-width(240px);
+    }
+  }
+
+  /*.action-col{
+    @include utils.set-all-width(40px);
+  }*/
```
Comment (D13ce) [inline frontend/components/notifications/management/NotificationsManagementDashboards.vue:325]: **Nitpick:** Please remove.

**Review by D13ce — CHANGES_REQUESTED** [review]
Good step, a few adaptions will be required though, see below :point_down:

File: frontend/public/mock/notifications/managementDashboard.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "via_discord": false
+      },
+      "networks": [1]
+    },
+    {
+      "group_id": 123,
+      "dashboard_id": 222,
+      "dashboard_name": "My account dashboard",
+      "dashboard_type": "account",
+      "subscriptions": ["missed_attestation", "proposed_attestation"],
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/public/mock/notifications/managementDashboard.json:23]: will be adapted once we get the API definitions (so resolve for now)

File: frontend/stores/notifications/useNotificationsManagementDashboardStore.ts
Comment (D13ce) [reply frontend/stores/notifications/useNotificationsManagementDashboardStore.ts:1]: Jeah, I like that. Thanks!

**Review by D13ce — APPROVED** [review]
LGTM

**Pull request merged**